pages/search/index.tsx

```diff
@@ -173,10 +216,13 @@ export default function Search({
            * TODO: The logic and copy for different scenarios will need to be added when
            * filters are implemented
            */
-          <Heading level="h3">No results. Try a different search.</Heading>
+          // @ts-expect-error
```

Comment:
Out of curiosity, what is the error?

Reply:
type error due to heading component not expecting tabindex. i added a comment.